gradle/compile-js-multiplatform.gradle

```diff
@@ -6,12 +6,20 @@ apply from: rootProject.file('gradle/node-js.gradle')
 
 kotlin {
     targets {
-        fromPreset(presets.js, 'js')
+        fromPreset(presets.js, 'js') {
+            // Enable built-in test runner only for IR target.
+            // These runners don't support changing js module name change.
+            if (js.hasProperty("irTarget")) {
+                irTarget.browser()
+                irTarget.nodejs()
```

Comment:
`mocha`/`jsdom` target is not supported?

Reply:
nodejs runs mocha. `jsdom` is not enabled yet, true. Would you need jsdom tests if browser tests will be enabled?

Reply:
I wouldn't if it worked like a charm on both teamcity and my local machine.
Please elaborate why these tests are enabled conditionally, while in atomicfu they are enabled unconditionally

Reply:
Apparently its not easy to enable browser tests on TC. Local browser tests pass for both backends. Currently `jsdom` tests are is disabled in IR backend only.